src/categories/categories.service.ts

```
@@ -0,0 +1,48 @@
+    return this.prisma.categories.findMany({
+      where: {
+        level: getCategoryDto.level,
+      },
+      include: {
+        children: true,
+      },
+    });
+  }
+
+  async findProducts(id: string, getCategoryProductDto: GetCategoryProductDto) {
+    const { page, perPage, sort, includeSubcategories } = getCategoryProductDto;
```

ESLint

[error] 'includeSubcategories' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)